### PR TITLE
More robust Raveboard server, GPT-3 support!

### LINKED ISSUE #155: Wildtalk supports GPT-3
Add GPT-3 API support to Wildtalk!

### PR DESCRIPTION
**Pull request by josephbirkner**
Fixes #155 and #154

### CODE REVIEW

File: modules/ravestate_gpt3/__init__.py [hunk 1]
```diff
@@ -14,14 +14,18 @@
 PROMPT_CONVERSATION_ROBOY = "Roboy: "
 PROMPT_CONVERSATION_HUMAN = "Human: "
 PROMPT = f"""
-{PROMPT_CONVERSATION_HUMAN} How are you?
-{PROMPT_CONVERSATION_ROBOY} I'm doing great!
-{PROMPT_CONVERSATION_HUMAN} What is your name?
-{PROMPT_CONVERSATION_ROBOY} My name is Roboy. I'm a robot boy!
-{PROMPT_CONVERSATION_HUMAN} Where are you from?
-{PROMPT_CONVERSATION_ROBOY} I am from Munich!
+The following is a conversation between a human and a young Robot called "Roboy". He is curious, cheeky and nerdy:
```
Comment (missxa) [inline modules/ravestate_gpt3/__init__.py:17]: would be good to have it defined in the yaml config

File: modules/ravestate/causal.py [hunk 2]
```diff
@@ -337,25 +337,36 @@ def activated(self, act: IActivation):
         """
         Called by activation which previously received a go-ahead
          from consent(), when it is truly proceeding with
-         running (after it got the go-ahead from all it's depended=on
+         running (after it got the go-ahead from all it's depended-on
```
Comment (missxa) [inline modules/ravestate/causal.py:340]:
```suggestion
         running (after it got the go-ahead from all its depended-on
```

File: modules/ravestate_gpt3/__init__.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    def gpt3(ctx: rs.ContextWrapper):
+        append_to_history(PROMPT_CONVERSATION_HUMAN, ctx[rawio.prop_in])
+        prompt = PROMPT + "\n".join(history + [PROMPT_CONVERSATION_ROBOY])
+        logger.info(prompt)
+        result = requests.post("https://api.openai.com/v1/engines/davinci/completions", json={
+            "prompt": prompt,
+            "temperature": 1.2,
+            "max_tokens": 64,
+            "top_p": 0.5,
+            "stop": ["\n"]
```
Comment (missxa) [inline modules/ravestate_gpt3/__init__.py:58]: these look like config file candidates to me too

Comment (josephbirkner) [reply modules/ravestate_gpt3/__init__.py:58]: True dat!

**Review by missxa — APPROVED** [review]

**Pull request merged**